api/Clipboard.json

```diff
@@ -395,7 +395,8 @@
               "version_added": "47"
             },
             "safari": {
-              "version_added": "13.1"
+              "version_added": "13.1",
+              "notes": "Must be called within user gesture event handlers like pointerdown or pointerup"
             },
             "safari_ios": {
               "version_added": "13.4"
```

Comment:
Could you apply the note here too? It's a near certainty that iOS has a similar user gesture requirement.